tests/phpunit/api/v3/PaymentProcessorTest.php

```diff
@@ -79,6 +79,38 @@ public function testPaymentProcessorCreate() {
   }
 
   /**
+   * Update payment processor.
+   */
+  public function testPaymentProcessorUpdate() {
+    $params = $this->_params;
+    $result = $this->callAPISuccess('payment_processor', 'create', $params);
+    $this->assertNotNull($result['id']);
+
+    $updateParams = array(
+      'id' => $result['id'],
+      'name' => 'Update API Test',
```

Comment:
updating the name here would also replicate the issue and test CRM-20591. Didn't write `is_active` as we don't have the test class_name as a file in CRM_Core_Payment_APITest which then produces a fatal error on require_once.